src/main/java/com/amazonaws/encryptionsdk/keyrings/StandardKeyrings.java

```diff
@@ -53,4 +54,17 @@ public static Keyring rawAes(String keyNamespace, String keyName, SecretKey wrap
     public static Keyring rawRsa(String keyNamespace, String keyName, PublicKey publicKey, PrivateKey privateKey, String wrappingAlgorithm) {
         return new RawRsaKeyring(keyNamespace, keyName, publicKey, privateKey, wrappingAlgorithm);
     }
+
+    /**
+     * Constructs a {@code Keyring} which combines other keyrings, allowing one OnEncrypt or OnDecrypt call
+     * to modify the encryption or decryption materials using more than one keyring.
+     *
+     * @param generatorKeyring A keyring that can generate data keys. Required if childrenKeyrings is empty.
+     * @param childrenKeyrings A list of keyrings to be used to modify the encryption or decryption materials.
+     *                         Required if generatorKeyring is null.
+     * @return The {@link Keyring}
+     */
+    public static Keyring multi(Keyring generatorKeyring, List<Keyring> childrenKeyrings) {
```

Comment:
NitPick: It might be worth adding this method also:

```
public static multi(Keyring generatorKeyring, Keyring ... childrenKeyRings) {
   return multi(generatorKeyring, Arrays.toList(childrenKeyRings));
}
```

Reply:
Added